swarm/storage/mru/resource.go

```diff
@@ -525,63 +558,73 @@ func (h *Handler) Load(addr storage.Address) (*resource, error) {
 
 	// create the index entry
 	rsrc := &resource{}
-	rsrc.UnmarshalBinary(chunk.SData[8:])
-	rsrc.nameHash = ens.EnsNode(rsrc.name)
-	h.set(addr.Hex(), rsrc)
-	log.Trace("resource index load", "rootkey", addr, "name", rsrc.name, "namehash", rsrc.nameHash, "startblock", rsrc.startBlock, "frequency", rsrc.frequency)
+	rsrc.UnmarshalBinary(chunk.SData)
```

Comment:
this `[8:]` thing was driving me crazy, so now `UnmarshalBinary` takes care of it.

Reply:
`(chunk.SData+8)` <3